docs/run.rst

```diff
@@ -160,22 +173,22 @@ Solving problems
 
 Sometimes the data object will not have an "OK" status. In such case,
 it is helpful to be able to check what went wrong (and where).
-Method :any:`stdout <resdk.resources.Data.stdout>` for data objects
-can be really helpful in such cases - it saves the standard output of
-data object's algorithm and returns it as a string:
+The :any:`stdout <resdk.resources.Data.stdout>` method on data objects
+can help---it returns the standard output of the data object
```

Comment:
i think it is worth to make a distinction between process and data object - stdout is generated by process, not data object. I would rephrase as: "it returns the standard output of the commands that created the data object".